code/game/objects/items/weapons/storage/storage.dm

```diff
@@ -367,14 +367,14 @@
 		add_fingerprint(user)
 
 		if(!prevent_warning)
-			for(var/mob/M in viewers(user, null))
-				if (M == usr)
-					usr << "<span class='notice'>You put \the [W] into [src].</span>"
-				else if (M in range(1)) //If someone is standing close enough, they can tell what it is...
-					M.show_message("<span class='notice'>\The [user] puts [W] into [src].</span>")
-				else if (W && W.w_class >= 3) //Otherwise they can only see large or normal items from a distance...
-					M.show_message("<span class='notice'>\The [user] puts [W] into [src].</span>")
-
+			user.visible_message_stealth(\
+				"<span class='notice'>[user] tucks \the [W] in \the [src].",\

```

Comment:
Tuck is not really better than "put", and in fact is in many cases worse. You can't really tuck something into a duffel bag, for example. Put is a more neutral term and more well suited for such a diverse class.